### PR TITLE
 [stable/nginx-ingress]Add report node internal ip when host network was enabled

### PR DESCRIPTION
**Pull request by Aisuko**
Signed-off-by: Aisuko <urakiny@gmail.com>

#### What this PR does / why we need it:

When we want to deploy `nginx-ingress` into bare-metal and via the host network, everything is just ok, but after your `ingress` was deployed, you can not find the ingress IP.   So, the `reportNodeInternalIp` parameter can report all nodes' IP which was deployed `nginx-ingress`, this is a helpful parameter. With IP you can add <IP><ingress host > to your external DNS service like `CoreDNS` etc.  and then the `ingress` host which you added can work fine. 

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/nginx-ingress/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.6.18
+version: 1.6.19
```
Comment (ChiefAlexander) [inline stable/nginx-ingress/Chart.yaml]: Lets bump the minor version for this change to 1.7.0

File: stable/nginx-ingress/README.md [hunk 2]
```diff
@@ -56,6 +56,7 @@ Parameter | Description | Default
 `controller.hostNetwork` | If the nginx deployment / daemonset should run on the host's network namespace. Do not set this when `controller.service.externalIPs` is set and `kube-proxy` is used as there will be a port-conflict for port `80` | false
 `controller.defaultBackendService` | default 404 backend service; required only if `defaultBackend.enabled = false` | `""`
 `controller.dnsPolicy` | If using `hostNetwork=true`, change to `ClusterFirstWithHostNet`. See [pod's dns policy](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy) for details | `ClusterFirst`
+`controller.reportNodeInternalIp` | If deploy `nginx-ingress` on bare-metal and via hostNetwork, change to `reportNodeInternalIp.enabled` is true, the `ingress` could get nodes' internal IP which  was deployed `NGINX-INGRESS`,  the ip and `ingress host` can be add to your DNS service | false
```
Comment (ChiefAlexander) [inline stable/nginx-ingress/README.md]: The description doesn't make sense to me. Can we change it to:
```
If using `hostNetwork=true`, setting `reportNodeInternalIp=true`, will pass the flag `report-node-internal-ip-address` to nginx-ingress. This sets the status of all Ingress objects to the internal IP address of all nodes running the NGINX Ingress controller.
```

File: stable/nginx-ingress/values.yaml [hunk 3]
```diff
@@ -29,6 +29,11 @@ controller:
   # to keep resolving names inside the k8s network, use ClusterFirstWithHostNet.
   dnsPolicy: ClusterFirst
 
+  # Bare-metal considerations via the host network https://kubernetes.github.io/ingress-nginx/deploy/baremetal/#via-the-host-network
+  # Ingress status was blank because there is no Service exposing the NGINX Ingress controller in a configuration using the host network, the default --publish-service flag used in standard cloud setups does not apply
+  reportNodeInternalIp:
```
Comment (ChiefAlexander) [inline stable/nginx-ingress/values.yaml]: Let's change the value to be just `reportNodeInternalIp: false` without having enabled as a key.

**Review by ChiefAlexander — CHANGES_REQUESTED** [review]

File: stable/nginx-ingress/Chart.yaml (hunk 1 above)
Comment (Aisuko) [reply stable/nginx-ingress/Chart.yaml]: Changed version to 1.7.0

File: stable/nginx-ingress/values.yaml (hunk 3 above)
Comment (Aisuko) [reply stable/nginx-ingress/values.yaml]: Changed `reportNodeInternalIp` as a Boolean.

**Comment by ChiefAlexander** [comment]
Awesome, thanks for this!
/lgtm

**Pull request merged**